todo_app/app.py

```diff
@@ -4,43 +4,60 @@
 
 from todo_app.data.trello_items import get_to_do_items, add_item_to_list, change_list_of_item, get_board_list, list_name
 
-app = Flask(__name__)
-app.config.from_object(Config())
+from todo_app.data.view_model import ViewModel
+from todo_app.Item import Item
 
-@app.route('/')
+def create_app():
+    
+    app = Flask(__name__)
+    app.config.from_object(Config())
+    
+    # All the routes and setup code etc
+    # e.g.
+    # @app.route('/')
+    # def index():
+    #     ...
+    
 
-def index():
-    get_board_list()
-    data = get_to_do_items()
-    return render_template('index.html', data=data )
+    @app.route('/')
 
-@app.route('/add_new_item', methods=['post'])
-def add_new_item():
+    def index():
+        get_board_list()
+        items = get_to_do_items()
 
-    list_status = request.form['list_name']
-    item_name = request.form['item_title']
-    item_description = request.form['item_desc']
+        item_list: list[Item] = []
```

Comment:
It looks like this variable gets initialised and then ignored - can we get rid of it?